### PR TITLE
[chassis] Fixed critical process not correct for database-chassis docker

### PR DESCRIPTION
**Pull request by abdosi**
What I did:

Fixed critical process not correct for database-chassis docker

How I did:
```
Critical process for database-chassis is redis-chassis but critical_process contains hard-coded
to `redis` program always. Instead using jinja2 template to render
critical process list based on database docker type. redis-chassis for database-chassis docker and redis for regular database docker.
```

How I verify:
Manual Verification

### CODE REVIEW

**Review by anamehra — APPROVED** [review]

File: dockers/docker-database/critical_processes.j2 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{% if INSTANCES %}
+{%     for redis_inst, redis_items in INSTANCES.items() %}
+program:{{ redis_inst }}
```
Comment (judyjoseph) [inline dockers/docker-database/critical_processes.j2:3]: Instances I see there are 2 of them does that mean will add both to critical process list ? 

    "INSTANCES": {
        "redis":{
            "hostname" : "127.0.0.1",
            "port" : 6379,
            "unix_socket_path" : "/var/run/redis/redis.sock",
            "persistence_for_warm_boot" : "yes"
        },
        "redis_chassis":{
            "hostname" : "redis_chassis.server",
            "port": 6380,
            "unix_socket_path": "/var/run/redis-chassis/redis_chassis.sock",
            "persistence_for_warm_boot" : "yes"
        }
    },

Comment (abdosi) [reply dockers/docker-database/critical_processes.j2:3]: no, this take care of having correct instance

`  update_chassisdb_config -j $db_cfg_file_tmp -k -p $chassis_db_port`

**Review by judyjoseph — APPROVED** [review]

**Comment by abdosi** [comment]
@yxieca can you please help merge this.

**Comment by yxieca** [comment]
@abdosi why is this change only needed for 202205 branch?

**Comment by yxieca** [comment]
@abdosi why is this PR skipping 202211 branch?

**Comment by abdosi** [comment]
> @abdosi why is this PR skipping 202211 branch?

yes @yxieca  it is needed. I missed to add the label. Added the label now

**Pull request merged**